src/App/Markdown.php

```diff
@@ -21,4 +21,27 @@
 
 class Markdown
 {
+    public function __construct(Writer $argument1)
```

Comment:
change `$argument1` with appropriate name, like `$writer`, etc. Other methods arguments need to be updated too.